docs/Installation Guide.md

```diff
@@ -236,6 +236,13 @@ In addition to this, you would also have to populate the database for django-cit
 
     python manage.py migrate
     python manage.py cities_light
+    
+if ImportError: No module named cities_light ,this means cities_light has not installed properly.
+For this perform the following steps:
+
+    pip uninstall django-cities-light
```

Comment:
@Himanshi-Khandelwal You will not get the error if you have installed it from requirements.txt